.pug-lintrc.json

```diff
@@ -0,0 +1,16 @@
+{
+  "disallowTrailingSpaces": true,
+  "disallowMultipleLineBreaks": true,
+  "disallowTemplateString": null,
+  "disallowHtmlText": null,
+  "disallowAttributeConcatenation": true,
```

Comment:
can you arrange it in alphabetical order so that we wont end up adding duplicate rule?

Reply:
Done